lib/language.js

```diff
@@ -37,6 +37,8 @@ exports.errors = {
         min: 'must contain at least {{limit}} items',
         max: 'must contain less than or equal to {{limit}} items',
         length: 'must contain {{limit}} items',
+        ordered: 'at position {{pos}} fails because {{reason}}',
+        orderedLength: 'at position {{pos}} fails because array must contain {{limit}} items',
```

Comment:
must contain at most {{limit}} items
Avoids taking care of non-required values ;)